vite_rails/lib/vite_rails/tag_helpers.rb

```diff
@@ -79,6 +79,17 @@ def vite_image_tag(name, **options)
     image_tag(vite_asset_path(name), options)
   end
 
+  if defined?(Rails) && Rails.gem_version >= Gem::Version.new('7.1.0')
```

Comment:
That approach would have worked, but I'd like to [provide more information](https://github.com/ElMassimo/vite_ruby/pull/409/commits/f517b49a8fd8e220f8981cb265033378ec86db77) in case users see it in the docs and try to use it in an older version of Rails.